src/Components/SmartComponents/Modals/CvePairStatusModal.js

```diff
@@ -92,8 +88,12 @@ export const CvePairStatusModal = ({ cves, updateRef, inventories, hasDifferentS
     }
 
     function getCvestatus() {
-        return (cveList &&
-            (checkboxState && cveList[0].cve_status_id || cveList[0].status_id.toString())) || '0';
+        if (cveList) {
+            if (checkboxState && !hasDifferentStatus) { return cveList[0].cve_status_id;}
+            else if (hasDifferentStatus) {return '0';}
+            else if (!checkboxState && !hasDifferentStatus) {return cveList[0].status_id.toString();}
```

Comment:
why are you returning `toString()`. if is needed should you also return in line 92: ` cveList[0].cve_status_id` as well?